.github/workflows/build.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: |
           wget https://scrutinizer-ci.com/ocular.phar
-          php ocular.phar code-coverage:upload --format=php-clover coverage.clover
+          php ocular.phar code-coverage:upload --format=php-clover tests/_output/coverage.xml
```

Comment:
Is this change needed?

Reply:
I'm not sure why the above is not uploading the code coverage to scrutinizer.

Reply:
Are you sure coverage is generated?

Reply:
yes,

https://github.com/yii-extension/app/blob/9ef3fb769f94baba5c6e1c87d92af76423032832/.github/workflows/build.yml#L97